var/spack/repos/builtin/packages/wi4mpi/package.py

```diff
@@ -14,7 +14,7 @@ class Wi4mpi(CMakePackage):
     url = "https://github.com/cea-hpc/wi4mpi/archive/v3.4.1.tar.gz"
     maintainers = ["adrien-cotte", "marcjoos-cea"]
 
-    version("3.6.0", sha256="9b790ffd5b73933fa8c32adeb0c429341e12efe146493904d1d4f04be7cf74c9")
+    version("3.6.0", sha256="06f48bf506643edba51dd04bfdfbaa824363d28549f8eabf002b760ba516227b")
```

Comment:
Checksum verified. Do you know why it changed?

Reply:
I think @marcjoos-cea changed version number, `mpirun --version` returned `3.5.0` instead of `3.6.0`.
Not 100% sure.